cmd/crc/cmd/config/config.go

```diff
@@ -24,6 +24,7 @@ const (
 	HTTPSProxy           = "https-proxy"
 	NoProxy              = "no-proxy"
 	ProxyCAFile          = "proxy-ca-file"
+	EnableMonitoring     = "enable-monitoring"
```

Comment:
@praveenkumar enable should only be for cluster? Or should we be more descriptive here?

Reply:
You mean name it as `enable-cluster-monitoring` instead?

Reply:
@gbraad yes, it is cluster scope only, more descriptive mean explicitly mention cluster like @anjannath suggested?

Reply:
I don't see what kind of information `cluster` is adding. Is there an other kind of monitoring we would like to add ? 

Either we go with `monitoring` or `cluster-monitoring-operator`, but not half of the name.